crates/builder/tests/common.rs

```diff
@@ -61,16 +72,59 @@ impl PathExt for Utf8PathBuf {
         files.sort();
```

Comment:
Do the replace before the sort because otherwise the order won't be determinstic.